UPDATING.md

```diff
@@ -228,6 +228,14 @@ Now that the DAG parser syncs DAG permissions there is no longer a need for manu
 
 In addition, the `/refresh` and `/refresh_all` webserver endpoints have also been removed.
 
+### TaskInstances now *require* a DagRun
+
+Under normal operation every TaskInstance row in the database would have DagRun row too, but it was possible to manually delete the DagRun and Airflow would still execute the TaskInstances.
+
+In Airflow 2.2 we have changed this and now there is a database-level foreign key constraint ensuring that every TaskInstance has a DagRun row.
+
+Before updating to this 2.2 release you will have to manually resolve any inconsistencies (add back DagRun rows, or delete TaskInstances) if you have any "dangling" TaskInstance" rows.
+
```

Comment:
Should we also mention that `clean_tis_without_dagrun_interval` has been removed?

Reply:
Probably should yes (though having "extra" config items doesn't cause any harm)